services/leaderboard/src/main/java/io/openliberty/spacerover/leaderboard/rest/Leaderboard.java

```diff
@@ -148,44 +160,21 @@ public Response clear() {
 
 	}
 
-	@Consumes(MediaType.APPLICATION_JSON)
 	@Produces(MediaType.APPLICATION_JSON)
-	@APIResponse(responseCode = "200", description = "Deleted user entries from the leaderboard.")
-	@APIResponse(responseCode = "500", description = "Failed to delete user entry from the leaderboard.")
-	@Operation(summary = "Deletes a user's entries from the leaderboard.")
+	@APIResponse(responseCode = "200", description = "Successfully cleared the leaderboard.")
+	@APIResponse(responseCode = "500", description = "Failed to clear the leaderboard.")
+	@Operation(summary = "Clear the leaderboard entries from the database.")
 	@DELETE
-	@Path("/")
-	public Response remUser(PlayerID playerID) {
-		MongoCollection<Document> collection = db.getCollection(LEADERBOARD_COLLECTION_NAME);
-		String pID = playerID.getPlayer();
-		DeleteResult result = collection.deleteMany(eq("player", pID));
+	@Path("/_id")
+	public Response clearSingleEntryWithID(@QueryParam(LeaderboardConstants.QUERY_PARAM_ID) String documentID) {
```

Comment:
currently to delete, for example, leaderboard entry with id 1234, you'd need to do:
`DELETE /leaderboard/_id?_id=1234`

change to use path param instead of query param for _id. this will simplify the endpoint to:
`DELETE /leaderboard/1234`

```suggestion
	@Path("/{_id}")
	public Response clearSingleEntryWithID(@PathParam(LeaderboardConstants.QUERY_PARAM_ID) String documentID) {
```

Reply:
Unfortunately it looks like the swagger API doesn't allow you to specify the path param like it did query param: ![image](https://user-images.githubusercontent.com/26169178/168692324-a763cb16-f2fb-4733-97d5-3926dbcaafae.png)


Am i doing something wrong?

Reply:
ahh, your the QUERY_PARAM_ID value is "id", but the path variable is "_id".
try changing the path to `@Path("/{id}")`